reactioncommerce/reaction
# 2217 Add Product Name To Invoice

Resolves [2217](https://github.com/reactioncommerce/reaction/issues/2217)

**Testing**
- Clone this Branch and run on your local machine
- Log In as an Admin
- Open Dashboard and Click on Processing Orders
- Click on `Print Invoice` just below `Captured Total`
- Observe the `Product Name` in `Title` field

**Screenshot**
<img width="798" alt="screen shot 2017-05-09 at 12 17 40 pm" src="https://cloud.githubusercontent.com/assets/10584060/25848285/98fafe30-34b1-11e7-8a3d-3a3b1ac3f3df.png">
